3-reveal/migrations/4-FI/1-Thailand-2019/deploy/events_jurisdictions_materialized_view.psql

```diff
@@ -0,0 +1,32 @@
+-- Deploy reveal_fi_thailand_2019:events_jurisdictions_materialized_view to pg
+-- requires: reveal_transaction_tables:events
+-- requires: reveal_transaction_tables:plans
+-- requires: reveal_transaction_tables:goals
+-- requires: reveal_transaction_tables:actions
+-- requires: utils:refresh_mat_views
+
+BEGIN;
+
+SET search_path TO :"schema",public;
+
+CREATE MATERIALIZED VIEW IF NOT EXISTS events_jurisdictions_materialized_view
+AS
+SELECT
+    events.id AS events_id,
```

Comment:
small nitpick, lets make this "event_id" to match everything else --> no other fields are pluralized